Microsoft.Toolkit.Uwp.UI/Converters/DateTimeToStringConverter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ******************************************************************
```

Comment:
Well, actually I usually use this to comply the absence of the StringFormat Markup Extension. Usually in WPF you format data in XAML binding in that way, and honestly, I always though that formatting data is something more related to the View than the ViewModel, since is deal on how you present data and not how you stored it.
By the way this can be more useful if generalized to be used with any data type.
But I can also remove it if anyone doesn't feel the need of a StringFormat substitute.
I agree with you on classes not to be sealed, so I removed the sealed modifier.